config/features.yml

```diff
@@ -138,9 +138,6 @@ features:
   hca_log_form_attachment_create:
     actor_type: user
     description: Enable logging all successful-looking attachment creation calls to Sentry at info-level
-  cg1010_oauth_2_enabled:
```

Comment:
Reminder that this will have to be [removed from the db](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Removingold/unusedfeaturetoggles) for each stack via the rails console as well. Thanks!